tools/cldr/likelyLocalesAdditional.json

```diff
@@ -263,5 +263,62 @@
     "Gong": "ka-Gong-GE",
     "Hrkt": "ja-Hrkt-JP",
     "Latn": "en-Latn-US",
-    "Piqd": "tlh-Piqd"
-}
+    "Piqd": "tlh-Piqd",
+    "ar-DJ": "ar-Arab-DJ",
+    "bs-BA": "bs-Latn-BA",
+    "de-AT": "de-Latn-AT",
+    "de-LU": "de-Latn-LU",
+    "el-GR": "el-Grek-GR",
+    "en-AM": "en-Latn-AM",
+    "en-AZ": "en-Latn-AZ",
+    "en-CN": "en-Latn-CN",
+    "en-ET": "en-Latn-ET",
+    "en-GE": "en-Latn-GE",
+    "en-IS": "en-Latn-IS",
+    "en-JP": "en-Latn-JP",
+    "en-LK": "en-Latn-LK",
+    "en-MM": "en-Latn-MM",
+    "en-MX": "en-Latn-MX",
+    "en-MY": "en-Latn-MY",
+    "en-PH": "en-Latn-PH",
+    "en-PK": "en-Latn-PK",
+    "en-PR": "en-Latn-PR",
+    "en-RW": "en-Latn-RW",
+    "en-SD": "en-Latn-SD",
+    "en-TW": "en-Latn-TW",
+    "en-TZ": "en-Latn-TZ",
+    "es-CA": "es-Latn-CA",
+    "es-PH": "es-Latn-PH",
+    "es-US": "es-Latn-US",
+    "et-EE": "et-Latn-EE",
+    "fr-BE": "fr-Latn-BE",
+    "fr-CH": "fr-Latn-CH",
+    "fr-GQ": "fr-Latn-GQ",
+    "fr-DJ": "fr-Latn-DJ",
+    "fr-DZ": "fr-Latn-DZ",
+    "fr-LB": "fr-Latn-LB",
+    "fr-RW": "fr-Latn-RW",
+    "ha-NG": "fr-Latn-NG",
+    "it-CH": "it-Latn-CH",
+    "kk-KZ": "kk-Cyrl-KZ",
+    "ms-MY": "ms-Latn-MY",
+    "ms-SG": "ms-Latn-SG",
+    "nl-BE": "nl-Latn-BE",
+    "pa-IN": "pa-Guru-IN",
+    "pa-PK": "pa-Arab-PK",
+    "pt-GQ": "pt-Latn-GQ",
+    "ru-BY": "ru-Cyrl-BY",
+    "ru-GE": "ru-Cyrl-GE",
+    "ru-KG": "ru-Cyrl-KG",
+    "ru-UA": "ru-Cyrl-UA",
+    "sq-ME": "sq-Latn-ME",
+    "sv-FI": "sv-Latn-FI",
+    "sv-SE": "sv-Latn-SE",
+    "tr-AM": "tr-Latn-AM",
+    "tr-AZ": "tr-Latn-AZ",
+    "tr-CY": "tr-Latn-CY",
+    "uk-UA": "uk-Cyrl-UA",
+    "ur-IN": "ur-Arab-IN",
```

Comment:
Should this be ur-Deva-IN ??

Reply:
No. Arab is correct. `Deva` is for `mr` and  `hi' language.